### ISSUE TITLE
Il faut entrer nos informations de users dans Remboursements???

### ISSUE THREAD

**Issue by carapas** [ageg/webapp#9]
Je vien de remarquer que le users doit entrer ses informations personnels dans remboursements... Pourtant toutes ses informations se retrouve dans req.session.userInfo.

**Comment by carapas** [comment]
Également dans 
var RequestSchema = new Schema({
  cip: String,
  prenom: String,
  nom: String,
  email: String,
  reference: Number,
  category: String,
  bill_count: Number,
  total: Number,
  notes: String
});

sa devrait plutôt etre:

var RequestSchema = new Schema({
  user: ObjectId,
  reference: Number,
  category: String,
  bill_count: Number,
  total: Number,
  notes: String
});

**Comment by carboneater** [comment]
Ouais, j'devais faire un merge avec master pour régler ça. (D'où mon pull request initial)
Maintenant, quand le user fait sa demande, ses infos sont copiées de la base de données. Elles sont quand même visibles pour le user, mais il ne peut pas les changer pendant une demande.
(Si le user veut changer ses infos de contact (ex: cell), il devra le faire ailleurs...)

Pour ton deuxième point, j'suis pas encore assez à l'aise avec Mongo pour tester, mais je voulais que les données du user dans sa demande soient figées dans la demande.
Je vais y repasser bientôt...
(Mongo fait mal à mon esprit de RDBMS...)

**Comment by carapas** [comment]
Tu devrais enlever toute la section sur les données du user de ton formulaire. Et dans le back-end simplement ajouter l'Id du user actuelle dans la requête.

Pourquoi les données de l'utilisateur doivent être figées? Selon moi, si un usager change ses informations personnels ceux-ci devrait également être updater dans la demande. 

Tu pourrait aller chercher les informations du user avec la rêquete suivante:
Request.findOne({_id: id}, function (err, demande){
  User.findOne({_id: demande.user}, (err, user){
  });
});

**Comment by carboneater** [comment]
Je suis d'accord que si le user met ses infos à jour, elles devraient se mettre à jour dans une demande active, mais pour les archives, stocker les informations de manière statique m'avait l'air d'être la meilleure option.

Pour le stockage des données, j'y reviendrai quand je serai rendu-là.
Mais je garde tes suggestions en tête.

**Comment by carapas** [comment]
Si on veut garder l'information statique je suggère de faire un schema RequestArchive ou les requêtes compléter pourrait être déplacer. Cependant, je ne vois pas vraiment pourquoi garder les informations statiques dans les archives est important. Les informations du user ne sont pas critiques et nous sommes certains qu'au final le cip ne changera pas!
